pom.xml

```diff
@@ -137,6 +137,11 @@
                 <artifactId>plexus-utils</artifactId>
                 <version>3.1.1</version>
             </dependency>
+            <dependency>
+                <groupId>ch.qos.logback</groupId>
+                <artifactId>logback-classic</artifactId>
+                <version>1.2.3</version>
+            </dependency>
         </dependencies>
     </dependencyManagement>
 
```

Comment:
Many dependencies appear to be outsince of the dependency management tag - is this right?

Reply:
Yes. 
We start using it recently. DependencyManagement should be used carefully. It helps with the control of versions of artifacts used in transitive dependencies.